### ISSUE TITLE
Can not use sklearnserver

### ISSUE THREAD

**Issue by jiaqianjing** [kubeflow/kfserving#1174]
/kind bug

**What steps did you take and what happened:**
Firstly, sklearn and sklearnserver has been installed. (see that https://github.com/kubeflow/kfserving/tree/master/python/sklearnserver).
but when I execute following command
```
python -m sklearnserver --model_dir model.joblib --model_name svm
```
I can get this message:
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.6/runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.6/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/home/jovyan/kfserving/python/sklearnserver/sklearnserver/__init__.py", line 16, in <module>
    from .sklearn_model_repository import SKLearnModelRepository
  File "/home/jovyan/kfserving/python/sklearnserver/sklearnserver/sklearn_model_repository.py", line 16, in <module>
    from kfserving.kfmodel_repository import KFModelRepository, MODEL_MOUNT_DIRS
ModuleNotFoundError: No module named 'kfserving.kfmodel_repository'
```

**Environment:**
kubeflow 1.10

**Comment by wengyao04** [comment]
KFModelRepository is introduced for multi-model and is not in kfserving 0.4.1, it will be released in 0.4.2. My suggestion is that
1) follow [kfserving installation](https://github.com/kubeflow/kfserving/blob/master/python/kfserving/README.md) 
2) install sklearnserver

**Comment by jiaqianjing** [comment]
> KFModelRepository is introduced for multi-model and is not in kfserving 0.4.1, it will be released in 0.4.2. My suggestion is that
> 
> 1. follow [kfserving installation](https://github.com/kubeflow/kfserving/blob/master/python/kfserving/README.md) to install kfserving first
> 2. install sklearnserver

3Q.
